.ci/docker/common/install_cuda.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-NCCL_VERSION=v2.21.5-1
```

Comment:
Please note. This may need to be broken into 2 parts. Docker update and Workflows and scripts update